components/usage/cmd/run.go

```diff
@@ -37,7 +37,7 @@ func run() *cobra.Command {
 				log.WithError(err).Fatal("Failed to get config. Did you specify --config correctly?")
 			}
 
-			err = server.Start(cfg)
+			err = server.Start(cfg, Version)
```

Comment:
I see that this `Version` is set during build, but how and where it is set and what is the format of the version string?

Reply:
I've observed either `commit-<hash>` or `main.<build-id>`

It's set by the CI job when it builds the images